dtests/src/test/java/io/snappydata/hydra/installJar/dynamicJarLoadingUsingSnappyShell.conf

```diff
@@ -14,6 +14,14 @@ INCLUDE $JTESTS/io/snappydata/hydra/cluster/startEmbeddedModeCluster.conf;
 INITTASK    taskClass   = io.snappydata.hydra.installJar.DynamicJarLoadingTest taskMethod  = HydraTask_executeInstallJarCommand_DynamicJarLoading
             threadGroups = snappyThreads;
 
+INITTASK       taskClass   = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSnappyJob
```

Comment:
Just to identify such type of issue in future quickly.
This will execute the static java snappy job by which we can identify that there are no issues in the job if it runs successfully.